clang/lib/3C/DeclRewriter.cpp

```diff
@@ -632,8 +641,17 @@ bool FunctionDeclBuilder::VisitFunctionDecl(FunctionDecl *FD) {
   // Combine parameter and return variables rewritings into a single rewriting
   // for the entire function declaration.
   std::string NewSig = "";
+  if (RewriteGeneric) {
+    NewSig += "_Itype_for_any(T";
```

Comment:
Per my earlier comment -- this should be `_For_any` if the generic parameter is not an itype.